nix-lib/default.nix

```diff
@@ -115,6 +116,22 @@ let
     then name
     else "@${scope}/${name}";
 
+  # Helper function to convert from a SPDX license string as found
+  # in package.json (https://docs.npmjs.com/files/package.json#license)
+  # to a nix license attrSet as found in lib.licenses.
+  #
+  # Implementation is taken from yarn2nix-moretea.
+  # It does not support SPDX expression syntax.
+  # https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix#L35-L42
+  spdxLicense = licstr:
```

Comment:
I’d call it something more longform, like `npmLicenseToNixpkgsSpdxLicense`

Reply:
:+1: